CHANGELOG.md

```diff
@@ -23,19 +41,13 @@ This document outlines major changes between releases.
 - google.golang.org/grpc dependency from v1.62.0 to v1.62.1 (#811)
 - github.com/minio/sio dependency from v0.3.0 to v0.4.0 (#811)
 
-### Updating from 0.30.1
+### Fixed
+- Save hashless object header in SpitV2 (#974)
+- Clean actual object version (#970)
 
-Notice that the configuration parameter `fschain.endpoints` contains multiple endpoints.
```

Comment:
Why is this removed?

Reply:
That’s an excellent question. It was rebased without conflict. Reverted